dracut/modules.d/90wwinit/parse-wwinit.sh

```diff
@@ -6,10 +6,14 @@
 if [ "${root}" = "wwinit" ]
 then
     info "root=${root}"
-    export wwinit_container=$(getarg wwinit.container=); info "wwinit.container=${wwinit_container}"
-    export wwinit_system=$(getarg wwinit.system=); info "wwinit.system=${wwinit_system}"
-    export wwinit_runtime=$(getarg wwinit.runtime=); info "wwinit.runtime=${wwinit_runtime}"
-    export wwinit_kmods=$(getarg wwinit.kmods=); info "wwinit.kmods=${wwinit_kmods}"
+    wwinit_uri="$(getarg wwinit.uri)"
+    export wwinit_container="${wwinit_uri}container&compress=gz"; info "winit_container=${wwinit_container}"
```

Comment:
should `info "winit_container` be `info "wwinit_container"` missing `w` in `winit` ?

Reply:
Thanks. Fixed.